src/System.Text.RegularExpressions/tests/RegexMatchTests4.cs

```diff
@@ -9,337 +9,129 @@
 public partial class RegexMatchTests
 {
     /*
-    Tested Methods:
-
-        public static Match Match(string input);     Noncapturing group : Actual - "(a+)(?:b*)(ccc)"
-            "aaabbbccc"
-
-        public static Match Match(string input);     Zero-width positive lookahead assertion: Actual - "abc(?=XXX)\\w+"
-            "abcXXXdef"
-
-        public static Match Match(string input);     Zero-width negative lookahead assertion: Actual - "abc(?!XXX)\\w+"
-            "abcXXXdef" - Negative
-
-        public static Match Match(string input);     Zero-width positive lookbehind assertion: Actual - "(\\w){6}(?<  =  XXX ) def "
-            " abcXXXdef "
-        public  static  Match  Match ( string  input ) ; Zero-width  negative  lookbehind  assertion :  Actual - " ( \ \ w ) { 6 } ( ? < ! XXX ) def "
-            " XXXabcdef "
-        public  static  Match  Match ( string  input ) ; Nonbacktracking  subexpression :  Actual - " [ ^ 0 - 9 ] + ( ?>[0-9]+)3"
-            "abc123"
-
+    public Match Match(string input);
+        - Noncapturing group: Actual - "(a+)(?:b*)(ccc)", "aaabbbccc"
+        - Zero-width positive lookahead assertion: Actual - "abc(?=XXX)\\w+", "abcXXXdef"
+        - Zero-width negative lookahead assertion: Actual - "abc(?!XXX)\\w+", "abcXXXdef" - Negative
+        - Zero-width positive lookbehind assertion: Actual - "(\\w){6}(?<  =  XXX ) def ", " abcXXXdef "
+        - Zero-width  negative  lookbehind  assertion: Actual - " ( \ \ w ) { 6 } ( ? < ! XXX ) def ", " XXXabcdef "
+        - Nonbacktracking subexpression: Actual - " [ ^ 0 - 9 ] + ( ?>[0-9]+)3", "abc123"
```

Comment:
Should we split the single test method below into one test for each of these, named appropriately?